example/src/main/java/org/secondbase/example/main/HelloSecondBase.java

```diff
@@ -15,8 +15,11 @@ private void printVariable() {
     }
 
     public static void main(final String[] args) throws SecondBaseException {
+        final String[] realArgs = new String[] {
+                "--service", "test"
+        };
         final HelloSecondBase helloSecondBase = new HelloSecondBase();
-        new SecondBase(args, new Flags().loadOpts(helloSecondBase));
+        new SecondBase(realArgs, new Flags().loadOpts(helloSecondBase));
```

Comment:
Ok. But it's still confusing to have a flag here which is not a part of the example, and the flag in the example is never used since the main args isn't parsed.